### DISCUSSION TITLE
Microsoft To Do App

### DISCUSSION THREAD

**Discussion by Cotrox** [zepp-health/zeppos-forum#71] [General]
Hi guys, i think i’m not too skilled right now ahah but it will be amazing to have the ability to sync microsoft to do lists. Do you think someone would work on it?

**Comment by leober-ramos33** [comment]
I don't think it's possible, but you can develop a simple TODO Android app that saves the TODO list to a file, and then integrate into a Zepp OS mini app to talk to the device and get that file saved by your application on the phone, and thus synchronize the list.

**Comment by wyliletv** [comment]
Microsoft ToDo would be amazing to have.
